modules/admin_manual/pages/installation/manual_installation/manual_installation_apache.adoc

```diff
@@ -124,3 +124,16 @@ sudo service apache2 reload
 
 If you want to use a threaded MPM, look at a FastCGI configuration where PHP is running
 in its own memory space. ownCloud limit´s its support to Apache prefork only.
+
+[NOTE]
+====
+In case you have enbaled `mpm_event` during an earlier setup of Apache, you may get conflicts notes from Apache. Use the following commands to solve this issue. The order of commands is recommended to ensure a smooth transition. 
```

Comment:
```suggestion
In case you have enabled `mpm_event` during an earlier setup of Apache, you may get conflict notes from Apache. Use the following commands to solve this issue. The order of commands is recommended to ensure a smooth transition. 
```